content/attacks/posts/2022-02-02-Wormhole.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2022-02-02
+custodians: Wormhole
+categories: Custodian
+title: Wormhole lost $325 million due to hacker attack
+---
+
+## Summary
+
+On February 2, 2022, Wormhole, a decentralized finance platform, fell prey to the big cryptocurrency theft. The attacker stole $325 million worth of crypto assets during the incident. Jump Crypto, parent company of the developers of the Wormhole, stepped in to replenish the stolen funds. Also, Wormhole offered white hat agreement and $10 million bug bounty to the hacker, with the expectation that attacker would return the funds. However, it seems that the anticipated resolution never materialized.
+
+## Attackers
+
+In order to execute the attack, the perpetrator successfully forged a legitimate signature for a transaction, granting them the ability to mint wETH, which represents an equivalent of Ethereum on the Solana blockchain, without providing an equivalent amount initially. The minted wETH was subsequently converted to Ethereum and transferred from Wormhole to the hackers' account, resulting in the liquidation of a substantial portion of the platform's Ethereum funds that were being held as collateral for transactions on the Solana blockchain.
+
+## Losses
+
+Wormhole lost 120,000 ETH, that was equal to $325 million. Also, as a result of the attack, the value of the Solana cryptocurrency experienced a decline of 10 percent. The funds were returned to customers the next day from Jump Crypto's assets.
+
+## Timeline
+
+- **February 2, 2022:** Wormhole upload on their Github disguised security fix. Most likely, hacker notice that and start planning the attack. Just hour later the hacker exploit vulnerability in the Wormhole's bridge protocol and stole  $325 million.
+- **February 3, 2022:**  Token bridge is back up and Jump Crypto stepped in to replace the stolen funds.
+
+## Security Failure Causes
+
+- **Weakness in bridge protocol:** Wormhole bridge protocol had a vulnerability on the Solana side that allowed attacker mint wETH.
+- **Lack of the security awareness:** Wormhole team know about weakness in their bridge and the necessary code to fix this vulnerability had been written. But the code was uploaded to the Github repository before it was applied to the production app. This fact allows a hacker to successfully exploit the protocol vulnerability.
```

Comment:
The Wormhole team knew about weaknesses in their bridge but the necessary code to fix it had been written.

This fact allowed...